source/quick-start-sinatra/download-and-install.txt

```diff
@@ -0,0 +1,111 @@
+.. _mongoid-qs-download-and-install:
+
+====================
+Download and Install
+====================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: ruby framework, odm, code example
+
+Prerequisites
+-------------
+
+To create the Quick Start application by using Sinatra, you need the
+following software installed in your development environment:
+
+- `Ruby <https://www.ruby-lang.org/en/documentation/installation/>`__
+- `RubyGems package manager <https://rubygems.org/pages/download>`__
+- A terminal app and shell. For MacOS users, use Terminal or a similar app.
+  For Windows users, use PowerShell.
+
+Download and Install the Mongoid Gem
+------------------------------------
+
+Complete the following steps to install and add the {+odm+} gem
+to your web application.
+
+.. procedure::
+   :style: connected
+
+   .. step:: Install {+odm+}
+
+      Ensure that the version of {+odm+} you install is compatible with the
+      version of {+language+} installed on your operating system. To
+      learn which versions are compatible, see the
+      :ref:`mongoid-compatibility` page.
+
+      Run the following command to install {+odm+}:
+
+      .. code-block:: bash
+
+         gem install mongoid
+
+      In {+language+}, packages are called **gems**.
+
+      When the installation completes, the command outputs the following
+      message:
+
+      .. code-block:: none
+         :copyable: false
+
+         Successfully installed mongoid-{+full-version+}
+         Parsing documentation for mongoid-{+full-version+}
+         Installing ri documentation for mongoid-{+full-version+}
+         Done installing documentation for mongoid after 1 seconds
+         1 gem installed
+
+   .. step:: Create a Quick Start application directory
+
+      Run the following commands to generate a new directory
+      called ``{+quickstart-sinatra-app-name+}`` and enter it:
+
+      .. code-block:: bash
+
+         mkdir {+quickstart-sinatra-app-name+}
+         cd {+quickstart-sinatra-app-name+}
+
+   .. step:: Create a ``Gemfile`` and add gems
+
+      All {+language+} applications must have a ``Gemfile`` that lists
+      the required gems. Run the following command to create a
+      ``Gemfile`` in your application:
+
+      .. code-block:: bash
+
+         touch Gemfile
+
+      Paste the following content into your ``Gemfile`` to add the
+      required gems:
+
+      .. code-block:: ruby
+
+         source 'https://rubygems.org'
+         
+         gem 'sinatra'
+         gem 'mongoid'
+         gem 'rackup'
+
+      The ``rackup`` gem provides a web server interface for your
+      application.
+
+   .. step:: Install gems
+
+      Run the following command to install the list of gems into your
+      application:
+
+      .. code-block:: bash
+
+         gem install bundler
+         bundle install
+
+      When the command runs successfully, the output in your
+      shell contains a ``Bundle complete!`` message.
```

Comment:
Q: Is "Bundle complete!" just part of the message? If so, this makes sense; otherwise, you could say "displays a...message"

Reply:
The message has info about the number of gems installed and some other info, which is why I phrased it this way. I think pasting the full message might be confusing as the number of installed packages could differ between users.